charts/portal/values-production.yaml

```diff
@@ -743,10 +743,17 @@ ingress-nginx:
 # in any production environment.
 # MySQL Stable Chart values - https://github.com/bitnami/charts/tree/master/bitnami/mysql
 mysql:
-  imageTag: "8.0.22-debian-10-r75"
+  image:
+    tag: "8.0.22-debian-10-r75"
   auth:
     username: portal
     existingSecret: database-secret
+  # podSecurityContext:
```

Comment:
podSecurity is under primary
https://github.com/bitnami/charts/blob/master/bitnami/mysql/values.yaml